tests/python/sdk/test_jobs.py

```diff
@@ -148,6 +149,34 @@ def test_can_export_dataset(
             request=request,
             cloud_storages=cloud_storages,
         )
+    
+    @pytest.mark.parametrize("format_name", ("CVAT for images 1.1",))
```

Comment:
- It makes sense to test this with all formats, not only one
- This functionality if not specific to SDK, so it's better to move it either into server API tests or server unit tests.